api/common/interfaces.go

```diff
@@ -570,7 +570,7 @@ func getAddressByName(s Serverer, params map[string]interface{}) map[string]inte
 		return respPacking(INTERNAL_ERROR, err.Error())
 	}
 
-	scriptHash, err := program.CreateRedeemHash(pubKey)
+	scriptHash, err := program.CreateProgramHash(pubKey)
```

Comment:
should we also rename this `scriptHash` to `programHash`?

Reply:
ok,I will